### PR TITLE
clean DefaultStatTags

### PR DESCRIPTION
**Pull request by zirain**
**Please provide a description of this PR:**

start from 1.17, stats implement in standard way, these lines are useless.

cc @kyessenov

### CODE REVIEW

**Review by kyessenov — CHANGES_REQUESTED** [review]
We still have to support extraStatTags because users should be able to run their Wasm filters with tagged metrics, but we can change the defaults to drop istio_stats tags.

File: tools/packaging/common/envoy_bootstrap.json [hunk 1]
```diff
@@ -96,12 +92,6 @@
         "tag_name": "mongo_prefix",
         "regex": "^mongo\\.(.+?)\\.(collection|cmd|cx_|op_|delays_|decoding_)(.*?)$"
       },
-      {{- range $a, $tag := .extraStatTags }}
```
Comment (kyessenov) [inline tools/packaging/common/envoy_bootstrap.json:99]: I'd suggest to keep it here, but change the default value to be empty.

Comment (zirain) [reply tools/packaging/common/envoy_bootstrap.json:99]: seems reasonable.

**Review by kyessenov — APPROVED** [review]

**Pull request merged**